src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Colors getRandomColor() {
+        Random colorRandom = new Random();
+        int result = colorRandom.nextInt(Colors.values().length);
+        return Colors.values()[result];
```

Comment:
```suggestion
        return Colors.values()[result].name();
```

![image](https://github.com/mate-academy/jv-lottery/assets/140905715/12747ca7-84b0-41fd-b141-5199bdf17c26)